twisted/pydoctor
# Have own Sphinx object inventory available at Sphinx build time.

Changes
=======

Create the Sphinx object inventory for each pydoctor target before all the plugins are loaded.
In this way, we have the object before the intersphinx loads.

Intersphinx plugin is designed to load the inventory from the source directory...but we want it from build.

Intersphinx is designed to use  absolute URL as it is used for external sites... but here we have an interrnal site link.


How to test
===========

```
rm -rf build/docs
tox -e doc
firefox build/docs/usage.html
```
Or on Read The Docs 
https://pydoctor--330.org.readthedocs.build/en/330/usage.html#building-pydoctor-together-with-sphinx-html-build

Check the link at `pydoctor.sphinx_ext.build_apidocs`  - It should be an absolute path to the site... but not an absolute URL.
